uat/features/mollom.feature

```diff
@@ -15,7 +16,8 @@ Feature: Mollom
     Then I should see "The form protection has been added"
 
 # This will fail until we figure out a way to pass private key
+  @api @dev @destructive
```

Comment:
Wonder if we should have something like a `@knownfail` tag for Scenarios that we know will fail.